tests/functional/workloads/cnv/test_cnv_node_replacement.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import pytest
+import random
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework.pytest_customization.marks import workloads, magenta_squad
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm, run_dd_io
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+from tests.functional.z_cluster.nodes.test_node_replacement_proactive import (
+    delete_and_create_osd_node,
+    select_osd_node_name,
+)
+from ocs_ci.helpers.helpers import (
+    verify_storagecluster_nodetopology,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
you may need to skip the test to run on external mode and baremetal

Reply:
Done